app/src/main/java/org/dhis2/usescases/eventsWithoutRegistration/eventDetails/ui/EventDetailsFragment.kt

```diff
@@ -228,6 +231,8 @@ class EventDetailsFragment : FragmentGlobalAbstract() {
     ) {
         Column {
             if (viewModel.getPeriodType() == null || (viewModel.getPeriodType() != null && viewModel.getPeriodType() == PeriodType.Daily)) {
+                Spacer(modifier = Modifier.height(16.dp))
```

Comment:
Same, avoid using spacer and use the vertical arragement in the Column composable